src/site/stages/build/drupal/api.js

```diff
@@ -105,6 +108,61 @@ function getDrupalClient(buildOptions) {
       throw new Error(`HTTP error: ${response.status}: ${response.statusText}`);
     },
 
+    /**
+     * Fetches and untars the CMS export.
+     *
+     * @return {String} - The path to the untarred CMS export.
+     */
+    async fetchExportContent({ debug } = { debug: false }) {
+      /* eslint-disable no-console */
+      const say = debug ? console.log : () => {};
+
+      say(
+        chalk.green('Fetching content export from'),
+        chalk.blue.underline(contentExportUri),
+      );
```

Comment:
I think this is a clever technique to avoid a bunch of extra `if`s throughout the code